dubbo-monitor/dubbo-monitor-api/src/main/java/org/apache/dubbo/monitor/support/MonitorFilter.java

```diff
@@ -45,15 +45,28 @@
 
     private static final Logger logger = LoggerFactory.getLogger(MonitorFilter.class);
 
+    /**
+     * The Concurrent counter
+     */
     private final ConcurrentMap<String, AtomicInteger> concurrents = new ConcurrentHashMap<String, AtomicInteger>();
 
+    /**
+     * The MonitorFactory
+     */
     private MonitorFactory monitorFactory;
 
     public void setMonitorFactory(MonitorFactory monitorFactory) {
         this.monitorFactory = monitorFactory;
     }
 
-    // intercepting invocation
+    /**
+     * The invocation interceptor,it will collect the invoke data about this invocation and send it to monitor center

```

Comment:
Do you feel we should add the importance of `Constants.MONITOR_KEY` here, because the method looks for this and take call based on that.

Reply:
+1 But I don't think it blocks this pull request. Maybe we can improve it later.